repository/src/main/java/org/apache/atlas/repository/store/graph/v2/preprocessor/AuthPolicyPreProcessor.java

```diff
@@ -238,6 +244,16 @@ public void processDelete(AtlasVertex vertex) throws AtlasBaseException {
         }
     }
 
+    private void authorizeDeleteAuthPolicy(AtlasEntity policy) throws AtlasBaseException {
+        if (getPolicyCategory(policy).equals(POLICY_CATEGORY_BOOTSTRAP) && getPolicySubCategory(policy).equals(POLICY_SUB_CATEGORY_COLLECTION)) {
```

Comment:
If a member want to delete just collection bootstrap policies will it fail or succeed?

Reply:
It should fail

Reply:
It will be allowed as Authcheck is skipped